pyo3-ffi/src/object.rs

```diff
@@ -337,6 +376,15 @@ extern "C" {
 // Flag bits for printing:
 pub const Py_PRINT_RAW: c_int = 1; // No string quotes etc.
 
+#[cfg(all(Py_3_12, not(Py_LIMITED_API)))]
+pub const _Py_TPFLAGS_STATIC_BUILTIN: c_ulong = 1 << 1;
+
+#[cfg(all(Py_3_12, not(Py_LIMITED_API)))]
+pub const Py_TPFLAGS_MANAGED_WEAKREF: c_ulong = 1 << 3;
```

Comment:
Is this intentionally equal to `Py_TPFLAGS_MANAGED_DICT`?

Reply:
`MANAGED_DICT` should be `1 << 4`, great catch 👍